ConsistencyManager/ConsistencyManager.swift

```diff
@@ -416,12 +416,13 @@ public class ConsistencyManager {
         if garbageCollectionInterval > 0 {
             // We're going to use a low priority queue for this timer
             // We need to use a dispatch queue because NSOperationQueue doesn't support delays
-            let dispatchQueue = dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_LOW, 0)
+            
+            let dispatchQueue = DispatchQueue.global(qos: DispatchQoS.QoSClass.default)
```

Comment:
seems like `DISPATCH_QUEUE_PRIORITY_LOW` has become `.utility` in swift 3. I will correct that.